### DISCUSSION TITLE
Thanks for making this

### DISCUSSION THREAD

**Discussion by Interpause** [tryolabs/norfair#115] [General] [answered]
I have been searching quite hard for an easy to use object tracker implementation in Python, especially one that is generalized/modular. Thanks for making this so I didn't have to resort to coding my own. I really agree with the part about most models being in a research state... struggled with that firsthand attempting to convert all the models I used to ONNX, and their pre & post-processing steps to numpy...

**Comment by joaqo** [comment, marked as answer]
Thank you very much, this is very kind of you!
